plugins/zlux-agent/doc/swagger/fileapi.yaml

```diff
@@ -169,14 +169,168 @@ paths:
       consumes:
         - application/json
       produces:
-        - text/json
+        - application/json
       responses:
         '200':
           description: successful operation
           schema:
             type: string
         '404':
           description: File could not be opened or does not exist
+    put:
+      summary: Creates a dataset
+      tags:
+        - Dataset
+      parameters:
+        - name: dataset
+          in: path
+          description: Cache results for future
+          required: true
+          type: string
+        - name: body
+          in: body
+          required: true
+          schema:
+            type: object
+            properties:
+              dsorg:
+                type: string
+                enum: [PS, PO]
+              blksz:
+                type: number
+              lrecl:
+                description: Desired record length
+                type: number
+              volser:
+                type: string
+              recfm:
+                type: string
+                enum: [A, B, F, U, V]
+              blkln:
+                description: The average data block length
+                type: number
+              ndisp:
```

Comment:
I think normal termination disposition for a data set deserves a description (because I've been in the industry for a few years now and I didn't know what it was)

Reply:
I wish I knew what it meant, too.